Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_rifle_nt.yml

```
@@ -83,17 +83,94 @@
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-1
+      map: ["enum.GunVisualLayers.Mag"]
+
+- type: entity
+  parent: MagazineBoxHeavyRifleNT
+  id: MagazineBoxHeavyRifleNTPractice
+  name: ammunition box (7,62mm NT practice)
+  description: Cardboard box full of 7.62x51mm full-power rifle rounds. Dated, but still widely used by marksman rifles and medium machine guns. Intended to hold non-harmful chalk ammunition.
+  components:
+  - type: BallisticAmmoProvider
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)